charts/rmd/values.yaml

```
@@ -84,6 +84,7 @@ rmd:
     emailPassword: ~
     # -- Folder name on the SMTP server
     emailFolder: "INBOX"
+  existingSecret: ""
```

```suggestion
  # -- Use an existing secret of creating one with this chart
  existingSecret: ""
```
(you would also need to rerun helm-docs if you merge this, it changes the README.md)